frontend/src/pages/Dashboard/components/SummaryBand/SummaryBand.module.scss

```
@@ -1,19 +1,14 @@
 // demo/layout_v3.html §10
 .summary {
   display: grid;
-  grid-template-columns: repeat(4, minmax(0, 1fr)) 300px;
+  // 목표 카드는 숫자·금액·막대·바닥 줄을 다 담으므로 30% 를 주고, 남은 70% 를
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)